app/assets/config/manifest.js

```diff
@@ -2,3 +2,4 @@
 //= link_tree ../images
 //= link application.js
 //= link application.css
+//= link_directory ../javascripts .js
```

Comment:
I added this to fix my `Rails: Sprockets::Rails::Helper::AssetNotPrecompiled` issue

https://stackoverflow.com/questions/35683185/rails-sprocketsrailshelperassetnotprecompiled-in-development/64203218#64203218

Reply:
hmm okidoke